packages/integration-oauth/src/providers/auth0.ts

```diff
@@ -0,0 +1,95 @@
+import { createUrl, handleRequest, authorizationHeaders } from "../request.js";
+import { scope, provider } from "../core.js";
+
+import type { Auth } from "lucia-auth";
+import type { OAuthConfig } from "../core.js";
+
+const PROVIDER_ID = "auth0";
+
+type Config = OAuthConfig & {
+	baseUrl: string;
```

Comment:
Feels like something like `appDomain` or something along those line makes more sense, unless `baseUrl` is the term used by Auth0?

Reply:
Term of Auth0 is domain. I will go with your suggestion and name it appDomain since it is the domain of a specific Auth0 app.